File: myBatis/src/main/java/com/xqy/www/mybatis/dao/AppTab.java
package com.xqy.www.mybatis.dao;

import javax.persistence.*;

@Table(name = "app_tab")
public class AppTab {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    @Column(name = "c_name")
    private String cName;

    @Column(name = "c_tab_id")
    private String cTabId;

    /**
     * @return id
     */
    public Integer getId() {
        return id;
    }

    /**
     * @param id
     */
    public void setId(Integer id) {
        this.id = id;
    }

    /**
     * @return c_name
     */
    public String getcName() {
        return cName;
    }

    /**
     * @param cName
     */
    public void setcName(String cName) {
        this.cName = cName;
    }

    /**
     * @return c_tab_id
     */
    public String getcTabId() {
        return cTabId;
    }

    /**
     * @param cTabId
     */
    public void setcTabId(String cTabId) {
        this.cTabId = cTabId;
    }
}

File: myBatis/src/main/kotlin/com/xqy/www/mybatis/dao/AppTab.kt
package com.xqy.www.mybatis.dao

import java.io.Serializable
import javax.persistence.*

//@Table(name = "app_tab")
//data class AppTab(@Id
                 // @GeneratedValue(strategy = GenerationType.IDENTITY) val id:Int, val name:String):Serializable



@Table(name = "`app_tab`")
class AppTab {
    /**
     * @return id
     */
    /**
     * @param id
     */
    @Id
    @Column(name = "`id`")
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Int? = null

    /**
     * @return name
     */
    /**
     * @param name
     */
    @Column(name = "`name`")
    var name: String? = null
}